### PR TITLE
Add Xcode 11 to requirements section

### LINKED ISSUE #133: [xcode 10] Build crashes
# Bug

Build crashes on IOS, because [labelColor variable](https://github.com/react-native-community/datetimepicker/blob/66be0879c324a69d250d5e6067b0f69683901f80/ios/RNDateTimePickerManager.m#L51) does not exist.

Actual error message:
```
.../node_modules/@react-native-community/datetimepicker/ios/RNDateTimePickerManager.m:51:33: 
error: no known class method for selector 'labelColor'
        defaultColor = [UIColor labelColor];
```

## Environment info

Library version: 2.3.0
IOS: 13.2.2

### PR DESCRIPTION
**Pull request by luancurti**
# Summary

Add Xcode 11 to requirements section because are some issues with build problems that can be fixed updating the Xcode

Closes #133 

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅❌     |
| Android |    ✅❌     |

## Checklist

- [ ] I have tested this on a device and a simulator
- [x] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [ ] I updated the typed files (TS and Flow)
- [ ] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ React Native date & time picker component for iOS and Android
     - [Android](#android)
   - [Running the example app](#running-the-example-app)
 
+## Requirements
+
+- Xcode 11 with iOS 13 SDK
```
Comment (vonovak) [inline README.md:57]:
```suggestion
- Xcode >= 11; please open a PR if you need to support building with older Xcode versions - see #133. Please note that we support iOS >= 10 but you need Xcode 11 to build.
```

Comment (luancurti) [reply README.md]: Thanks!

File: README.md [hunk 2]
```diff
@@ -52,6 +52,10 @@ React Native date & time picker component for iOS and Android
     - [Android](#android)
   - [Running the example app](#running-the-example-app)
 
+## Requirements
+
+- Xcode >= 11; please open a PR if you need to support building with older Xcode versions - see #133. Please note that we support iOS >= 10 but you need Xcode 11 to build.
```
Comment (vonovak) [inline README.md:57]:
```suggestion
- Xcode >= 11; please open a PR if you need to support building with older Xcode versions - see [#133](https://github.com/react-native-community/datetimepicker/issues/133). Please note that we support iOS >= 10 but you need Xcode 11 to build.
```

the link actually needs to be there, sorry :D

Comment (luancurti) [reply README.md]: Done

**Comment by vonovak** [comment]
great, thanks!

**Pull request merged**